internal/agent/config/config.go

```diff
@@ -409,6 +419,11 @@ func mergeConfigs(base, override *Config) {
 	overrideIfNotEmpty(&base.TPM.DevicePath, override.TPM.DevicePath)
 	overrideIfNotEmpty(&base.TPM.StorageFilePath, override.TPM.StorageFilePath)
 
+	// audit log
```

Comment:
not super falimiler with overrideifnotempty function but shouldnt we just use it here ?